### ISSUE TITLE
find: ‘/tmp/[...]/cp_font_files.sh’: Permission denied

### ISSUE THREAD

**Issue by Chiitoo** [Winetricks/winetricks#1264]
Having '/tmp' mounted with 'noexec' will lead into issues when installing things such as fonts:

```
$ WINEPREFIX="/mnt/wine/tricks/tmp" winetricks corefonts
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20190615-next - sha256sum: 5bcdadda63aa02330e33861197bfbcb3d72711ca6ca1f5d87e03f6953ca764d8 with wine-4.11 and WINEARCH=win64
Executing w_do_call corefonts
Executing load_corefonts
Executing w_do_call andale
Executing load_andale
Executing cabextract -q -d /mnt/wine/tricks/tmp/dosdevices/c:/windows/temp/_andale /home/winetrickster/.cache/winetricks/corefonts/andale32.exe
find: ‘/tmp/winetricks.NyLDHNGn/w.winetrickster.25683/cp_font_files.sh’: Permission denied
------------------------------------------------------
Running /usr/bin/wineserver -w. This will hang until all wine processes in prefix=/mnt/wine/tricks/tmp terminate
------------------------------------------------------
Executing wine regedit C:\windows\Temp\_andale\_register-font.reg
wine: cannot find L"C:\\windows\\system32\\winemenubuilder.exe"
000b:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (2).
Executing wine64 regedit C:\windows\Temp\_andale\_register-font.reg
Executing cp /mnt/wine/tricks/tmp/dosdevices/c:/windows/temp/_andale/_register-font.reg /tmp/winetricks.NyLDHNGn/_reg_90ad6a9f_25683.reg
Executing wine regedit C:\windows\Temp\_andale\_register-font.reg
Executing wine64 regedit C:\windows\Temp\_andale\_register-font.reg
Executing cp /mnt/wine/tricks/tmp/dosdevices/c:/windows/temp/_andale/_register-font.reg /tmp/winetricks.NyLDHNGn/_reg_f6204652_25683.reg
------------------------------------------------------
andale install completed, but installed file /mnt/wine/tricks/tmp/dosdevices/c:/windows/Fonts/andalemo.ttf not found
------------------------------------------------------
```

**Comment by austin987** [comment]
@Chiitoo could you try https://github.com/austin987/winetricks/commit/bbea13c03a25c6bace148cddbfe655cbc6dd3f14 ?

That fixes allfonts for me; I still need to do some more tests before pushing.

**Comment by Chiitoo** [comment]
Seems to make things work here too so far.

Thanks!

**Comment by austin987** [comment]
Fixed by c03c13a3209a43e825636621b99e027c441fcaae
